charts/kubearchive/templates/operator/operator.yaml

```diff
@@ -48,34 +48,59 @@ spec:
             capabilities:
               drop:
                 - ALL
-        - args:
+        - name: manager
+          {{- if .Values.operator.debug }}
+          args:
+            - --listen=:40000
+            - --headless=true
+            - --accept-multiclient
+            - --api-version=2
+            - --log
+            - exec
+            - /ko-app/operator
+            - --
+            - --health-probe-bind-address=:8081
+            - --metrics-bind-address=127.0.0.1:8080
+            - --leader-elect
+          command:
+            - ./go/bin/dlv
+          {{- else }}
+          args:
             - --health-probe-bind-address=:8081
             - --metrics-bind-address=127.0.0.1:8080
             - --leader-elect
           command:
             - /ko-app/operator
+          {{- end}}
           env:
             - name: KUBEARCHIVE_NAMESPACE
               valueFrom:
                 fieldRef:
                   fieldPath: metadata.namespace
             {{- include "kubearchive.v1.otel.env" . | nindent 12 }}
           image: {{ .Values.operator.image }}
+          {{- if not .Values.operator.debug }}
           livenessProbe:
             httpGet:
               path: /healthz
               port: {{ $port }}
             initialDelaySeconds: 15
             periodSeconds: 20
-          name: manager
+          {{- end }}
```

Comment:
If I understand this correctly it adds the livenessProbe when the operator is not on debug mode. Why we don't want the livenessProbe when debugging? It causes the pod to restart because the debugger is blocking the execution of the probe?

Reply:
Yes, with this change the livenessProbe won't be available in debug mode.

I added this because we could potentially be debugging the code in `main.go` before the manager server starts